crates/bevy_render/src/view/window/mod.rs

```diff
@@ -456,18 +456,13 @@ pub fn create_surfaces(
             .surfaces
             .entry(window.entity)
             .or_insert_with(|| {
-                let surface_target = SurfaceTargetUnsafe::RawHandle {
-                    raw_display_handle: window.handle.display_handle,
-                    raw_window_handle: window.handle.window_handle,
-                };
                 // SAFETY: The window handles in ExtractedWindows will always be valid objects to create surfaces on
```

Comment:
Does this safety comment apply to `window.handle.get_handle()`? It originally justified just `create_surface_unsafe`.

Reply:
Right. Updated the safety comment to just mention the main thread restriction.